api/AudioParam.json

```diff
@@ -334,7 +336,9 @@
               "version_added": "14"
             },
             "chrome_android": {
-              "version_added": "18"
+              "version_added": "18",
+              "partial_implementation": true,
+              "notes": "This sets the target volume at the specified time, but it doesn’t ramp to it, causing this function to behave like <code>setValueAtTime()</code>."
```

Comment:
If an implementation doesn't ramp, it's a bug.

Reply:
@rtoy Yes, and in BCD, we mention bugs in the form of notes, just like we’re doing.  If that bug causes an issue that affects compatibility, such as this one, we mark Partial Implementation as true, as I’ve stated a couple of times.  🙂

Reply:
Fair enough. I have no further objections.